Facebook API/src/facebookAPI/graphAPITest.java

```diff
@@ -0,0 +1,24 @@
+package facebookAPI;
+
+import static org.junit.Assert.*;
+import java.util.Date;
+import org.junit.Test;
+import com.mashape.unirest.http.exceptions.UnirestException;
+
+public class graphAPITest {
+	GraphAPI API = new GraphAPI("CAACEdEose0cBAPO6slZCnZCHwdXcDboWv8wU4pS7UvuiSWrB5pMTv8hRntMei2bibEZCwLZA3ePla172xnyr0ZCBJst4FGLpJ4z6GC8ZCIK8iIjCBasPSPyZCQvZAZCjbRHHLWLrobc2lmmblFJ6cHgKefYLfnSQJWRZCkKofyolYToZBMZBWGJB1a6TnPdvOr9vZBfMFrzjBN712r2TirqY3O2Eh");
+	
+	@Test
+	public void nameTest() throws UnirestException {
```

Comment:
Names of test methods should say what they do.  Like "testsGetName" or something